Zend/Optimizer/zend_func_infos.h

```diff
@@ -222,7 +222,6 @@ static const func_info_t func_infos[] = {
 	F1("mb_strrichr", MAY_BE_STRING|MAY_BE_FALSE),
 	F1("mb_substr", MAY_BE_STRING),
 	F1("mb_strcut", MAY_BE_STRING),
-	F1("mb_strimwidth", MAY_BE_STRING),
```

Comment:
May I please ask the reason for removing this hint? I would like to understand.

Reply:
This file is generated from the stub file. Basically it contains information about RC count the value will have that the function returns. This information is then used in the optimizer, I'm assuming to elide some reference counting operations. But I haven't dug into the code myself.

Reply:
Hmm. OK.